moodle/moodle.go

```diff
@@ -716,7 +721,30 @@ func (c *Controller) createService(foo *operatorv1.Moodle) string {
 
 	return servicePort
 }
-
+func (c *Controller) ensurePluginsInstalled(pod *apiv1.Pod) string {
+	fmt.Println("EnsurePluginsInstalled")
+	ownerRef1 := pod.OwnerReferences
+	ownerRef := metav1.GetControllerOf(pod)
```

Comment:
Rather using GetControllerOf, I would suggest using a direct approach like you are doing in line 728. Did that not work?

Reply:
Right, I did try this at one point.I had problems besides this, it didn't get to the print at the start of the function because I got this error message, I narrowed the error down a little

 `E0416 20:14:26.316818       1 controller.go:342] pod 'namespace1/moodle1' in work queue no longer exists`